akka-actor/src/main/scala/akka/actor/Deployer.scala

```diff
@@ -244,7 +244,8 @@ private[akka] class Deployer(val settings: ActorSystem.Settings, val dynamicAcce
   def lookup(path: Iterable[String]): Option[Deploy] = deployments.get().find(path)
 
   def deploy(d: Deploy): Unit = {
-    @tailrec def add(path: Array[String], d: Deploy, w: WildcardIndex[Deploy] = deployments.get): Unit = {
+    @tailrec def add(path: Array[String], d: Deploy): Unit = {
```

Comment:
`Recursive call used default arguments instead of passing current argument values`
are manually fixed, take from #28991